cmd/state-custom-installer/docker/centos-8/runCompile.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+cd cli
+git pull
+git checkout BE-1031-there-is-a-standalone-tool-for-unpacking-and-relocating-artifacts-in-installers 
```

Comment:
Same as above.

Reply:
This code was rolled into the Centos-7 code, and was deleted.